schedule/functional/create_hdd_transactional_server.yaml

```diff
@@ -0,0 +1,28 @@
+---
+name: create_hdd_transactional_server
```

Comment:
Please, no duplication, use the existing https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/schedule/yast/transactional_server/create_hdd_transactional_server.yaml which is maintained by Yast team.

Reply:
okay, I didn't notice this yaml file under yast, thanks

Reply:
hm, this yaml file is for sles, so without changes for TW it won't work.

Reply:
Yes.
Please, use conditional schedule.

Reply:
We can have separate schedule for TW, it's not a problem and will be cleaner this way.

Reply:
yes, this is easier to create and maintain as  well, but I spent already time to get this done :) thanks!